### DISCUSSION TITLE
Feature Request: Auto Video Subtitle Generation and Transcription using Whisper OpenAI

### DISCUSSION THREAD

**Discussion by Riteshp2001** [huynhsontung/Screenbox#332] [Ideas]
Currently, ScreenBox doesn't have built-in capabilities for automatic subtitle generation and transcription. This functionality would be incredibly valuable for a wide range of users and use cases. if its possible it would be awesome feature to be an add-on

**Comment by United600** [comment]
The [platform](https://support.microsoft.com/en-us/windows/use-live-captions-to-better-understand-audio-b52da59c-14b8-4031-aeeb-f6a47e6055df) already provides a pretty good alternative.

**Reply by Riteshp2001** [reply]
yea but better added into player would be awesome thing as an overlay on the ongoing video

**Comment by alchemist-al** [comment]
Something like VL Sub to search for subtitles would be great.
